script/SpokeDeployer.s.sol

```diff
@@ -108,9 +107,6 @@ contract SpokeDeployer is CommonDeployer {
         // Rely async requests manager
         globalEscrow.rely(address(asyncRequestManager));
 
-        // Rely sync requests manager
-        asyncRequestManager.rely(address(syncRequestManager));
```

Comment:
Not needed anymore since this was only used for the `syncManager.addVaults` call